src/webhooks/webhooks.ts

```diff
@@ -1,9 +1,22 @@
 import { SignatureVerificationException } from '../common/exceptions';
 import { deserializeEvent } from '../common/serializers';
 import { Event, EventResponse } from '../common/interfaces';
+import {
+  CryptoProvider,
+  SubtleCryptoProvider,
+  NodeCryptoProvider,
+} from '../common/crypto';
 
 export class Webhooks {
-  private encoder = new TextEncoder();
+  private cryptoProvider: CryptoProvider;
+
+  constructor(subtleCrypto?: typeof crypto.subtle) {
+    if (typeof crypto.subtle !== 'undefined') {
```

Comment:
just double checking, will this be `undefined` as a string?

Reply:
Yup, because `typeof` always returns the string representation of the type.